www/pfSense-pkg-squid/files/usr/local/pkg/squid_auth.xml

```diff
@@ -180,6 +180,18 @@
 				<option><name>3</name><value>3</value></option>
 			</options>
 		</field>
+		<field>
+                        <fielddescr>Transport</fielddescr>
+			<fieldname>ldap_urltype</fieldname>
+			<type>select</type>
+			<default_value>standard</default_value>
+			<options>
+				<option><name>TCP - Standard</name><value>standard</value></option>
+				<option><name>TCP - STARTTLS</name><value>starttls</value></option>
+				<option><name>SSL - Encrypted</name><value>ssl</value></option>
+			</options>
+			<description>If option 'SSL Encrypted' or 'TCP - STARTTLS' is used, you need to add the CA certificate of your AD to the Operating System Trust Store. pfSense 2.5 only feature.</description>
```

Comment:
This still needs updated:
> If option 'SSL Encrypted' or 'TCP - STARTTLS' is used, you need to add the CA certificate of your AD to the Operating System Trust Store. pfSense 2.5 only feature.

Change to:
> If 'SSL Encrypted' or 'TCP - STARTTLS' is selected, the CA certificate of the LDAP server must be trusted by the Operating System Trust Store. This is automatic for certificates signed by globally trusted CAs such as Let's Encrypt; self-signed CAs can optionally be added to the Trust Store on pfSense 2.5.

Reply:
done